packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -127,6 +127,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           <Label className={ css('ms-Dropdown-label') } id={ id + '-label' } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
         ) }
         <div
+          { ...divProps }
```

Comment:
instead of moving divProps all the way up to the top, how about just moving className below divProps. often users want to override the role or an aria value, and putting divProps up here won't allow them to do that.